lib/openstudio-workflow.rb

```diff
@@ -153,3 +153,73 @@ def load_adapter(name, adapter_options = {})
     end
   end
 end
+
+# Extend OS Runner to persist measure information throughout the workflow
+class ExtendedRunner < OpenStudio::Ruleset::OSRunner
+  # Allow former arguments to be set and read
+  #todo: Consider having the former arguments passed in in initialization, and define as attr_reader
+  attr_accessor :former_workflow_arguments
```

Comment:
@rHorsey SWIG provides a method for overriding C++ methods in Ruby called directors.  I am not sure if this works with Ruby's accessors, I assume it would not.  Try implementing your own method:

    def former_workflow_arguments
      ...
    emd